apps/web/components/auth/SignupForm.tsx

```diff
@@ -27,14 +28,17 @@ export const SignupForm = () => {
     if (!isValid) {
       return;
     }
+
     setSigningUp(true);
     try {
-      await createUser(
+      const user = await createUser(
         e.target.elements.name.value,
         e.target.elements.email.value,
         e.target.elements.password.value,
         inviteToken
       );
+      const teamId = user.memberships[0].teamId;
+      addDemoProduct(teamId);
```

Comment:
I think it would be better if we move the demo data creation call into the `api/v1/users` route (createUser) to always create a demoProduct when a new user / team is created.